src/cleanvision/utils/utils.py

```diff
@@ -50,7 +50,7 @@ def get_filepaths(
         Sorted list of image filepaths, note that all paths in this list are absolute paths
     """
 
-    abs_dir_path = os.path.abspath(dir_path)
+    abs_dir_path = os.path.abspath(os.path.expanduser(dir_path))
```

Comment:
No.
It is for expanding paths like `~/data` to `/home/sanjana/data`, otherwise it builds a weird path like `/home/sanjana/cleanvision/~/data` and throws an error.